GEOSldas_App/ldas_setup

```diff
@@ -151,15 +168,94 @@ class LDASsetup:
+                self.rqdExeInp['ENSEMBLE_FORCING']     = 'YES' 
+                self.nens                              = 32           # for now, hardwire nens for atm ens
+            else :
+                exit("Error. Unknown value of self.ladas_cpl.\n")
+            self.rqdExeInp['NUM_LDAS_ENSEMBLE']        = self.nens
+            self.first_ens_id                          = 1
+            self.rqdExeInp['FIRST_ENS_ID']             = self.first_ens_id 
+            self.rqdExeInp['EXP_DOMAIN']               = 'CF'+ self.agcm_res +'x6C_GLOBAL' 
+            
+            # when coupled to ADAS, "BCS_PATH" EXCLUDE bcs version info  
+            # hard-wired BCS_PATH for now 
+            self.rqdExeInp['BCS_PATH'] = "/discover/nobackup/projects/gmao/bcs_shared/fvInput/ExtData/esm/tiles"
+            self.rqdExeInp['BCS_PATH']                 = self.rqdExeInp['BCS_PATH'].rstrip('/') + '/' + self.bcs_version 
+            if self.bcs_version == "Icarus-NLv3" :
+                self.rqdExeInp['BCS_PATH']             = self.rqdExeInp['BCS_PATH'] + '_new_layout'  
+            self.rqdExeInp['BCS_RESOLUTION']           = 'CF'+ self.agcm_res +'x6C_CF'     + self.agcm_res +'x6C'
+            self.rqdExeInp['RESTART_DOMAIN']           = 'CF'+ self.agcm_res +'x6C_GLOBAL'
+           
+            # the following are not in default rqdExeInp list; hardwire for now
+            self.rqdExeInp['LAND_ASSIM']               = "YES"
+            self.rqdExeInp['MET_HINTERP']              = 0
+            self.rqdExeInp['LANDASSIM_DT']             = "10800"
+            self.rqdExeInp['LANDASSIM_T0']             = "013000"
+            jsgmt1                                     = "00000000"
+            jsgmt2                                     = hours_to_hhmmss(self.varwindow/60)       # convert minutes to HHMMSS
+            self.rqdExeInp['JOB_SGMT']                 = f"{jsgmt1} {jsgmt2}"
+            self.rqdExeInp['NUM_SGMT']                 = 1 
+            self.rqdExeInp['FORCE_DTSTEP']             = 3600
+
+            # determine END_DATE = BEG_DATE + TIME_STEP_OF_ADAS_CYCLE
+            _beg_date = datetime.strptime( self.rqdExeInp['BEG_DATE'], "%Y%m%d %H%M%S")
+            _hours    = int(self.rqdExeInp['JOB_SGMT'][ 9:11])
+            _end_date = _beg_date + timedelta(hours=self.varwindow/60) 
+
+            self.rqdExeInp['END_DATE']                 = _end_date.strftime("%Y%m%d %H%M%S") 
+            #default or input from command line 
+            self.rqdExeInp['NML_INPUT_PATH']           = "/discover/nobackup/qzhang/LDAS_INPUTS/SPECnml/" 
+            if self.LDAS_nml_path != 'None' :
+                self.rqdExeInp['NML_INPUT_PATH']       = self.LDAS_nml_path
+            self.rqdExeInp['MWRTM_PATH']               = "/discover/nobackup/qzhang/LDAS_INPUTS/MWRTM/" 
+            if self.mwrtm_path != 'None' :
+                self.rqdExeInp['MWRTM_PATH']           = self.mwrtm_path   
```

Comment:
I think we should drop the default values here.  If we hardwire these paths anywhere, it should be in fvsetup.

Reply:
dropped default values in ldas_setup.  default values in fvsetup